### DISCUSSION TITLE
linux aarch64 新版

### DISCUSSION THREAD

**Discussion by cstkingkey** [xfangfang/wiliwili#137] [General]
release 里没有aarch64版本了。 0.6提示更新，且没有声音。

**Comment by xfangfang** [comment]
声音相关的问题可以独立开一个issue来说。

有这么几个原因：
1. 之前的aarch64版本的只支持opengl，但是貌似很多arm处理器只支持opengl es，（存疑）我不确定是否真的有人在用。
2. 很多小白用户在下载steam deck客户端的时候会弄混下错

如果确实是在使用的话，我有时间可以编译一份放上来。

**Comment by xfangfang** [comment]
@cstkingkey 可以从这里下载：https://github.com/xfangfang/DIY/actions/runs/5071269149

**Reply by cstkingkey** [reply]
还是没有声音

**Reply by xfangfang** [reply]
> 还是没有声音

@cstkingkey 
前面有说，声音的问题可以单独发个issue。

linux下没有声音的原因有很多，比如是不是kde下把单个应用的声音关掉了，抑或是flatpak导致的神奇问题，这个问题应该不是应用本身的问题。

我的建议是从源码编译运行，查看播放视频时mpv的log。

因为Linux系统众多，每个人配置都是不同的，这个只能自己来解决了。
